lightly/api/api_workflow_predictions.py

```diff
@@ -62,17 +62,17 @@ def create_or_update_predictions(
         progress_bar: Optional[tqdm.tqdm] = None,
         max_workers: int = 8,
     ) -> None:
-        """Creates or updates the predictions for specific samples
+        """Creates or updates the predictions for specific samples.
 
         Args:
             sample_id_to_prediction_singletons
                 A mapping from the sample_id of the sample to its corresponding prediction singletons.
                 The singletons can be from different tasks and different types.
 
             prediction_version_id:
-                 A numerical id (e.g timestamp) to distinguish different predictions of different model versions.
-                 Use the same id if you don't require versioning or if you wish to overwrite the previous schema.
-                 This id must match the id of a prediction task schema.
+                A numerical ID (e.g timestamp) to distinguish different predictions of different model versions.
```

Comment:
Please update the `(e.g timestamp)` part to match the docstring from further above.